Makefile

```diff
@@ -17,3 +17,6 @@ test-e2e:
 clean:
 	rm -f cluster-samples-operator
 
+
+#FYI if we ever add back in git version into the binary, compile like
+#go build -ldflags '-X github.com/openshift/cluster-samples-operator/vendor/k8s.io/client-go/pkg/version.gitVersion=$(shell git describe) -X github.com/openshift/cluster-samples-operator/vendor/k8s.io/client-go/pkg/version.gitCommit=$(shell git rev-parse HEAD)' ./cmd/cluster-samples-operator
```

Comment:
i'd leave this comment up in the `build` target

Reply:
I had it there originally but then the comment was showing up in the `make build` output

didn't like that, so moved it here

Reply:
if you don't indent the comment it shouldn't show up.

Reply:
(I just tested this)

Reply:
thx part of next push